Bikeshed (remote) returns an error on main branch

```
$ make
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  335k    0   288  100  335k     32  38918  0:00:08  0:00:08 --:--:--    62

Error running preprocessor, returned code: 2.
LINK ERROR: No 'dfn' refs found for 'aborted flag' with for='['AbortSignal']'.
&lt;a data-link-for="AbortSignal" data-link-type="dfn" data-lt="aborted flag">aborted flag&lt;/a>
 ✘  Did not generate, due to fatal errors
make: *** [Makefile:5: remote] Error 22
```

Removing the offending link solves that problem:
```
--- a/fetch.bs
+++ b/fetch.bs
@@ -7476,7 +7476,7 @@ <h3 id=fetch-method>Fetch method</h3>
  <li><p>Let <var>request</var> be <var>requestObject</var>'s <a for=Request>request</a>.
 
  <li>
-  <p>If <var>requestObject</var>'s <a for=Request>signal</a>'s <a for=AbortSignal>aborted flag</>
+  <p>If <var>requestObject</var>'s <a for=Request>signal</a>'s aborted flag
   is set, then:
 
   <ol>
```
but perhaps the best fix is to add the missing definition instead.

--- a/fetch.bs
+++ b/fetch.bs
@@ -7476,8 +7476,8 @@ method steps are:
  <li><p>Let <var>request</var> be <var>requestObject</var>'s <a for=Request>request</a>.
 
  <li>
-  <p>If <var>requestObject</var>'s <a for=Request>signal</a>'s <a for=AbortSignal>aborted flag</a>
-  is set, then:
+  <p>If <var>requestObject</var>'s <a for=Request>signal</a> is <a for=AbortSignal>aborted</a>,
+  then:
 
   <ol>
    <li><p><a>Abort fetch</a> with <var>p</var>, <var>request</var>, and null.

